pkgs/development/python-modules/urwidgets/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, isort
+, setuptools
+, urwid
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "urwidgets";
+  version = "0.1.1";
+  pyproject = true;
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "AnonymouX47";
+    repo = "urwidgets";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-0aZLL0NutptPkuLHv3bTzR1/SNqLgMdUYWET6mLE0IU=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    urwid
+  ];
+
+  passthru.optional-dependencies = {
+    colors = [
+      isort
+    ];
+  };
+
```

Comment:
```suggestion
```
It seems that they want isort and its dependency, color, for development, and color is not optional-dependencies for this package.
https://github.com/AnonymouX47/urwidgets/blob/daa1d1a70bd7727d563cd56574aece537c36a85e/requirements.txt#L3-L4